main.go

```diff
@@ -128,6 +136,26 @@ func delete(cmd *cobra.Command, args []string) error {
 	return db.Delete(k)
 }
 
+func listDbs(cmd *cobra.Command, args []string) error {
+	db, err := openKV("")
+	if err != nil {
+		return err
+	}
+	db.Sync()
+  pn, err := db.Client().DataPath()
+  if err != nil {
+    return err
+  }
```

Comment:
Since you're not using the db from `openKV()`, you could use `client.NewClientWithDefaults()` here instead.

Reply:
Implemented in [37abcf9](https://github.com/charmbracelet/skate/pull/29/commits/37abcf9bba4644602963e908a214d9560eb1086a) :)